backend/src/routes/api/nb-events/eventUtils.ts

```diff
@@ -6,12 +6,16 @@ export const getNotebookEvents = async (
   namespace: string,
   nbName: string,
 ): Promise<V1Event[]> => {
-  const eventList = await fastify.kube.coreV1Api
-    .listNamespacedEvent(namespace, undefined, undefined, undefined, 'involvedObject.kind=Pod')
+  return fastify.kube.coreV1Api
+    .listNamespacedEvent(
+      namespace,
+      undefined,
+      undefined,
+      undefined,
+      `involvedObject.kind=Pod,involvedObject.name=${nbName}-0`,
+    )
```

Comment:
Can we add a reasonable `limit` ?

```listNamespacedEvent(namespace: string, pretty?: string, allowWatchBookmarks?: boolean, _continue?: string, fieldSelector?: string, labelSelector?: string, limit?: number, resourceVersion?: string, timeoutSeconds?: number, watch?: boolean, options?: 
```

Reply:
Ah yes, that's good... I wonder how limits work with sorting -- do we need to specify a sort on time?

Reply:
If you limit, you chop it off the end -- we'll need to paginate the results before calling the client -- which is sure to be less pressure on the list call, but not faster for the client-server calls. 

> limit – limit is a maximum number of responses to return for a list call. If more items exist, the server will set the `continue` field on the list metadata to a value that can be used with the same initial query to retrieve the next set of results. Setting a limit may return fewer than the requested amount of items (up to zero items) in the event all requested objects are filtered out and clients should only use the presence of the continue field to determine whether more results are available. Servers may choose not to support the limit argument and will return all of the available results. If limit is specified and the continue field is empty, clients may assume that no more results are available. This field is not supported if watch is true. The server guarantees that the objects returned when using continue will be identical to issuing a single list call without a limit - that is, no objects created, modified, or deleted after the first request is issued will be included in any subsequent continued requests. This is sometimes referred to as a consistent snapshot, and ensures that a client that is using limit to receive smaller chunks of a very large result can ensure they see all possible objects. If objects are updated during a chunked list the version of the object that was present at the time the first list result was calculated is returned.

Reply:
Yup, that's right, i tried limit but it was getting the oldest events, that's why i was trying to get some sort of filter by date but i didn't find anything relevant